lightly/api/utils.py

```diff
@@ -196,7 +196,7 @@ def get_api_client_configuration(
     token: Optional[str] = None,
     raise_if_no_token_specified: bool = True,
 ) -> Configuration:
-    host = getenv("LIGHTLY_SERVER_LOCATION", "https://api.lightly.ai")
+    host = getenv("LIGHTLY_SERVER_LOCATION", "https://api.lightly.ai").strip().rstrip("/")
```

Comment:
Can you move this to a separate function, e.g. `get_lightly_server_location_from_env` and test it properly please?

Reply:
Thanks, broken out, could you take another look at this?